doc/tutorials/error_analysis/error_analysis_part1.ipynb

```diff
@@ -0,0 +1,553 @@
+    "\n",
+    "There are several different ways to express the precision of your measurements. We will begin by briefly discussing what they are and what their differences are. After that, we will continue with the *standard error of the mean* as a viable option to be presented in your simulation results.\n",
+    "\n",
+    "### Standard deviation\n",
+    "The standard deviation is a measure for how much individual samples are expected to deviate from the mean. We want to use precise terminology, and therefore need to state that, in fact, we cannot directly measure the standard deviation but only estimate it. A commonly used estimator for the standard error is\n",
+    "\n",
+    "$\n",
+    "\\begin{align}\n",
+    "    \\bar\\sigma = \\sqrt{\\frac{1}{N-1}\\sum_{i=1}^{N}(x_i-\\bar{x})^2}\\tag{1}\n",
+    "\\end{align}\n",
+    "$\n",
+    "\n",
+    "where $\\bar\\sigma$ is the estimator of the standard deviation $\\sigma$, $N$ the number of samples, $x_i$ the individual samples and $\\bar{x}$ their mean. This estimator somewhat resembles the \"square root of the variance\". However, instead of dividing by $N$ as for the variance, we divide by $N-1$. This accounts for the fact that we do not know the true mean value, but only estimate it (by averaging over all samples); and therefore we have one additional degree of freedom. Another way of viewing the $N-1$ divisor is to understand that for a single measurement, it is impossible to define the variance, which is represented by a \"division by zero\".\n",
+    "\n",
+    "### Standard error of the mean\n",
+    "The standard error of the mean (often abbreviated as SEM) describes how much the mean value of your samples is expected to deviate from the true mean value. Imagine repeating the whole simulation over and over again, taking $N$ samples every time and averaging over them. The SEM quantifies how much those averages of $N$ samples will fluctuate around their mean. In fact, it is defined as the standard deviation of the average over $N$ samples.\n",
+    "\n",
+    "At first glance, it might seem to be very expensive to compute the SEM, because one would have to repeat the whole simulation many times. However, under the right circumstances, the SEM can be estimated from *a single series* of $N$ measurements. We will discuss how this can be done.\n",
+    "\n",
+    "### Confidence interval\n",
+    "A common confidence interval is the $95~\\%$ confidence interval. If we were given a number and a range around it, called the $95~\\%$ confidence interval, we would know that only $5~\\%$ of all samples are expected to lie outside that range. If the samples are normally distributed with a standard deviation $\\sigma$, the $95~\\%$ confidence interval would be roughly $\\pm1.96\\sigma$.\n",
```

Comment:
`95 % confidence interval, we would know that only 5 % of all samples are expected to lie outside that range`

I'm not exactly sure about this. In probability theory, <img src="https://render.githubusercontent.com/render/math?math=%5Cmu%20%5Cpm%201.96%5Csigma" alt="\mu \pm 1.96\sigma" height=18> excludes 5% of the probability distribution. However in statistics, the CI is an estimator of the lower and upper bounds of <img src="https://render.githubusercontent.com/render/math?math=%5Cmu" alt="\mu" height=13> for repeated experiments. I'm not convinced that the CI doubles as an estimator of the proportion of future samples to lie outside its range (assuming that by "expected", you mean "future").

Reply:
Good point, thanks!